### PR TITLE
VAGOV-6110: do not show facility main phone number if it is not there

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
## Description
gets rid of an error if a facility does not have a main phone

## Testing done
locally with staging data

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/static-pages/facilities/OtherFacilityListWidget.jsx [hunk 1]
```diff
@@ -59,17 +59,19 @@ export default class OtherFacilityListWidget extends React.Component {
...
-                  )}`}
-                >
-                  {facility.attributes.phone.main.replace(/[ ]?x/, '')}
-                </a>
-              </div>
+              {facility.attributes.phone.main && (
+                <div className="main-phone vads-u-margin-bottom--1">
+                  <strong>Main phone: </strong>
+                  <a
+                    href={`tel:${facility.attributes.phone.main.replace(
```
Comment (ElijahLynn) [inline src/applications/static-pages/facilities/OtherFacilityListWidget.jsx]: Is there a way to put a comment in here saying what this regex is attempting to do? Would make it easier for the next person.

Comment (ahay-agile6) [reply src/applications/static-pages/facilities/OtherFacilityListWidget.jsx]: comment added

**Review by ElijahLynn — APPROVED** [review]
:+1:

File: src/applications/static-pages/facilities/FacilityPhone.jsx [hunk 2]
```diff
@@ -15,31 +15,38 @@ export default class FacilityPhone extends React.Component {
...
-            )}
-          </a>
-        </div>
+        {this.props.facility.attributes.phone.main && (
+          <div className="main-phone vads-u-margin-bottom--1">
+            <strong>Main phone: </strong>
+            <a
+              href={`tel:${
+                // this replaces an "x" that appears from the api data with a space
+                this.props.facility.attributes.phone.main.replace(/[ ]?x/, '')
```
Comment (U-DON) [inline src/applications/static-pages/facilities/FacilityPhone.jsx]: I realize this existed before this PR, but I think this formatting should be relegated to a helper function. I guess `facilityUtilities.js` in the same directory would the be home for it. Also, does it replace with a space, or does it just remove the x completely?

```
// This removes (?) an "x" that appears from the API data.
const cleanPhoneNumber = number => number.replace(/[ ]?x/, '');
```

You can also save an extra replace each time you use it (also in the other component):

```
const { phone } = this.props.facility.attributes;
const mainPhone = cleanPhoneNumber(phone.main);
const mentalHealthClinicPhone = cleanPhoneNumber(phone.mentalHealthClinic);
// Use formatted numbers in the markup below.
```

**Review by U-DON — APPROVED** [review]

File: src/applications/static-pages/facilities/FacilityPhone.jsx (hunk 2 above)
Comment (ahay-agile6) [reply src/applications/static-pages/facilities/FacilityPhone.jsx]: i like the suggestion and implemented :smile:  thank you!

**Pull request merged**